lib/auth/storage/storage.go

```diff
@@ -235,22 +235,22 @@ func (p *ProcessStorage) WriteTeleportVersion(ctx context.Context, version *semv
 	return nil
 }
 
-func rdpLicenseKey(version uint32, issuer, company, productID string) backend.Key {
-	return backend.NewKey("rdplicense", issuer, strconv.Itoa(int(version)), company, productID)
+func rdpLicenseKey(key types.RDPLicenseKey) backend.Key {
+	return backend.NewKey("rdplicense", key.Issuer, strconv.Itoa(int(key.Version)), key.Company, key.ProductID)
 }
 
 type RDPLicense struct {
 	LicenseData []byte `json:"license_data"`
 }
 
 // WriteRDPLicense writes an RDP license to local storage.
-func (p *ProcessStorage) WriteRDPLicense(ctx context.Context, version uint32, issuer, company, productID string, license []byte) error {
+func (p *ProcessStorage) WriteRDPLicense(ctx context.Context, key types.RDPLicenseKey, license []byte) error {
```

Comment:
Suggestion: I would favor *types.RDPLicenseKey for less copying. Same for other params.